### PR TITLE
Document all the `/staging/{project_name}/excluded_requests` endpoints

### PR DESCRIPTION
**Pull request by saraycp**
With OpenAPI

### CODE REVIEW

File: src/api/public/apidocs-new/paths/staging_staging_workflow_project_excluded_requests.yaml [hunk 1]
```diff
@@ -0,0 +1,146 @@
+get:
+  summary: List excluded requests of a staging project
+  description: List the requests excluded from the staging project associated to the specified project
```
Comment (hennevogel) [inline src/api/public/apidocs-new/paths/staging_staging_workflow_project_excluded_requests.yaml:3]:
```suggestion
  description: List the requests excluded from the staging associated to the specified project
```

File: src/api/public/apidocs-new/paths/staging_staging_workflow_project_excluded_requests.yaml [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+                sumary: "Project project_name doesn't have an asociated Staging Workflow"
+            Unknown Project:
+              value:
+                code: unknown_project
+                summary: "Project not found: <some project>."
+  tags:
+    - Staging Workflow
+
+post:
+  summary: List the requests excluded from the current staging workflow
```
Comment (hennevogel) [inline src/api/public/apidocs-new/paths/staging_staging_workflow_project_excluded_requests.yaml:33]: `:post` is not list but add right?

File: src/api/public/apidocs-new/paths/staging_staging_workflow_project_excluded_requests.yaml [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+            Unknown Project:
+              value:
+                code: unknown_project
+                summary: "Project not found: <some project>."
+  tags:
+    - Staging Workflow
+
+delete:
+  summary: Delete the specified request from the list of excluded requests
+  description: Stop excluding the request from the staging workflow by deleting the specified request from the list of exluded requests
```
Comment (hennevogel) [inline src/api/public/apidocs-new/paths/staging_staging_workflow_project_excluded_requests.yaml:92]:
```suggestion
  description: Stop excluding the request from the staging workflow.
```

File: src/api/public/apidocs-new/paths/staging_staging_workflow_project_excluded_requests.yaml [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+  requestBody:
+    description: List of requests to be excluded from the staging workflow
```
Comment (hennevogel) [inline src/api/public/apidocs-new/paths/staging_staging_workflow_project_excluded_requests.yaml:98]:
```suggestion
    description: List of requests to stop excluding from the staging workflow
```

File: src/api/public/apidocs-new/components/schemas/staging/excluded_requests.yaml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+      type: object
+      properties:
+        id:
+          type: integer
+          example: 42
+          xml:
+            attribute: true
+        description:
+          type: string
+          example: Lorem ipsum dolor sit amet
```
Comment (hennevogel) [inline src/api/public/apidocs-new/components/schemas/staging/excluded_requests.yaml:15]: I guess a real world example would make things easier to understand here..

**Review by hennevogel — CHANGES_REQUESTED** [review]

File: src/api/public/apidocs-new/paths/staging_staging_workflow_project_excluded_requests.yaml (hunk 2 above)
Comment (saraycp) [reply src/api/public/apidocs-new/paths/staging_staging_workflow_project_excluded_requests.yaml]: copy + paste is evil. Fixed!

File: src/api/public/apidocs-new/paths/staging_staging_workflow_project_excluded_requests.yaml [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+      description: Not Found
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            project_without_staging_workflow:
+              value:
+                code: not_found
+                sumary: "Project project_name doesn't have an associated Staging Workflow"
```
Comment (saraycp) [inline src/api/public/apidocs-new/paths/staging_staging_workflow_project_excluded_requests.yaml:82]: This typo is fixed here (and all the endpoints in this PR), the text in the code will be fixed with https://github.com/openSUSE/open-build-service/pull/14221.

**Comment by saraycp** [comment]
I should use `project_name` to define all the staging-related endpoints. The filenames are going to change as well. I move this PR to draft until I adapt everything.

**Review by danidoni — APPROVED** [review]

**Pull request merged**